flink-kubernetes-shaded/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-kubernetes-operator-parent</artifactId>
+      <version>1.0-SNAPSHOT</version>
+      <relativePath>..</relativePath>
+    </parent>
+
+    <artifactId>flink-kubernetes-shaded</artifactId>
+    <name>Flink Kubernetes Shaded</name>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-kubernetes_${scala.version}</artifactId>
+            <version>${flink.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.code.findbugs</groupId>
+                    <artifactId>jsr305</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.flink</groupId>
+                    <artifactId>flink-shaded-force-shading</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Does this do anything? flink-kubernetes is a shaded jar it might not expose any deps

Reply:
Yes. This is unnecessary. I will fix it.

Reply:
Hmm. We might still need this.

```
[INFO] --- maven-dependency-plugin:3.1.1:tree (default-cli) @ flink-kubernetes-shaded ---
[INFO] org.apache.flink:flink-kubernetes-shaded:jar:1.0-SNAPSHOT
[INFO] \- org.apache.flink:flink-kubernetes_2.12:jar:1.14.3:compile
[INFO]    +- org.slf4j:slf4j-api:jar:1.7.15:compile
[INFO]    +- com.google.code.findbugs:jsr305:jar:1.3.9:compile
[INFO]    \- org.apache.flink:flink-shaded-force-shading:jar:14.0:compile
```

Reply:
we can also exclude */* for simplicity there